src/esn.calendar.libs/app/components/event/form/event-form.controller.spec.js

```diff
@@ -1941,6 +1941,51 @@ describe('The CalEventFormController controller', function() {
       });
     });
 
+    describe('open location link if is valid in new window', function() {
+      it('should return false if the location is not valid', function() {
+
+        scope.event = CalendarShell.fromIncompleteShell({
+          title: 'title',
+          path: '/calendars/' + owner._id + '/' + calendarTest.id + '/eventID',
+          start: moment('2016-12-08 12:30'),
+          end: moment('2016-12-08 13:30'),
+          location: 'http://123.com',
+          etag: '123456',
+          alarm: {
+            trigger: '-PT1M',
+            attendee: 'test@open-paas.org'
+          }
+        });
+        initController();
+        scope.editedEvent = scope.event.clone();
+
+        scope.openLink();
```

Comment:
```suggestion
```

Reply:
why deleting the call of open link ?

Reply:
the test is checking the `isValidURL`.
the `openLink` function call has no meaning here and it does nothing related to the test.